src/test/resources/application.properties

```diff
@@ -1,4 +1,4 @@
 spring.datasource.driver-class-name=org.h2.Driver
-spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1
+spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_ON_EXIT=FALSE
```

Comment:
Why you doubled the "DB_CLOSE(....)" phrase , and should't we modify the tests to check specify entities on "findAll" methods?

Reply:
Right - didn't notice I added it twice.

In this PR, I wanted to solve the error that pops up in CircleCi only.
I'll work on tests tomorrow if it will be possible. Now all the major changes are done, so now is the time to do it.